io.github.pineapple-pictures/linglong.yaml

```diff
@@ -0,0 +1,20 @@
+package:
+  id: io.github.pineapple-pictures
```

Comment:
I personally strongly suggest we could align to the AppStream component ID and desktop-id when possible. For this case, the application uses `net.blumia.pineapple-pictures` as both its AppStream component ID and desktop-id, instead of `io.github.pineapple-pictures`.

Reply:
get it！
Thank you for your open source project!